webhook_features/src/features/summary_comment.rs

```diff
@@ -188,6 +189,25 @@ async fn refresh_summary_comment(ctx: &Context, repo: Repository, pr_number: u64
 
     let cmt = util::get_metadata_sections_from_comments(&all_comments, pr_number);
 
+    let mut ignore_users: Vec<String> = vec![];
+    if let Some(cmt_id) = cmt.id {
+        let reactions = ctx.octocrab.all_pages(
+            ctx.octocrab
+                .issues(&repo.owner, &repo.name)
+                .list_comment_reactions(cmt_id.0)
```

Comment:
What is this `.0`? I presume it is because the type is wrong?

Reply:
Yes, thanks for working on my PR's followups. Removed the `.0`.